src/integration-test/utils/TestDiContainerProvider.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Container } from "inversify";
+import type { BaseIntegrationTestsConfig } from "../utils/imodels-client-test-utils/iModelsClientTestUtilsExports";
+import { TestUtilBootstrapper } from "./imodels-client-test-utils/TestUtilBootstrapper";
+import { testUtilTypes } from "./imodels-client-test-utils/TestUtilTypes";
+import { IModelsClientsTestsConfig } from "./IModelsClientsTestsConfig";
+import "reflect-metadata";
+
+let container: Container;
+export function getTestDIContainer(): Container {
+  if (container) {return container;}
```

Comment:
Curly brace formatting around the return statement is wrong.

Reply:
fixed (hopefully) all such cases